Tests/DangerTests/GitTests.swift

```diff
@@ -1,6 +1,130 @@
-import Danger
+@testable import Danger
+import DangerFixtures
 import XCTest
 
 final class GitTests: XCTestCase {
-    func test() {}
+    private var git: Git { githubFixtureDSL.git }
+
+    func testDecodesCommits() {
+        XCTAssertEqual(git.commits, [
+            Git.Commit(
+                sha: "93ae30cf2aee4241c442fb3242543490998cffdb",
+                author: Git.Commit.Author(name: "Ash Furrow", email: "ash@ashfurrow.com", date: "2016-07-26T19:54:16Z"),
+                committer: Git.Commit.Author(name: "Ash Furrow", email: "ash@ashfurrow.com", date: "2016-07-26T19:55:00Z"),
+                message: "[Xcode] Updates for compatibility with Xcode 7.3.1.",
+                parents: ["68c8db83776c1942145f530159a3fffddb812577"],
+                url: "https://api.github.com/repos/artsy/eidolon/commits/93ae30cf2aee4241c442fb3242543490998cffdb"
+            ),
+            Git.Commit(
+                sha: "4cf1e41f72516a4135f1738c47f7dd3d421ff3c4",
+                author: Git.Commit.Author(name: "Ash Furrow", email: "ash@ashfurrow.com", date: "2016-07-26T19:55:53Z"),
+                committer: Git.Commit.Author(name: "Ash Furrow", email: "ash@ashfurrow.com", date: "2016-07-26T19:55:53Z"),
+                message: "[CI] Updates Travis to Xcode 7.3.",
+                parents: ["93ae30cf2aee4241c442fb3242543490998cffdb"],
+                url: "https://api.github.com/repos/artsy/eidolon/commits/4cf1e41f72516a4135f1738c47f7dd3d421ff3c4"
+            ),
+            Git.Commit(
+                sha: "d0d72ec5b5ee90c2513a8aafb48911ae5bcdf4ac", author: Git.Commit.Author(name: "Ash Furrow", email: "ash@ashfurrow.com", date: "2016-07-26T21:17:40Z"),
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 14...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/GitTests.swift;
  Line: 27;
-->

- :warning: Line should be 140 characters or less: currently 169 characters (`line_length`)